tests/selenium/spec/table-of-contents-spec.js

```diff
@@ -13,14 +13,14 @@ describe('table of contents navigation spec', () => {
   const tocPage = new TableOfContentsPage('/test_page/');
 
   beforeEach(util.itHelper(async () => {
+    await tocPage.resizeXLarge();  // At smaller sizes, table of contents is hidden
     await tocPage.navigate('/test_page/');
     await tocPage.refresh();
-    await tocPage.resizeXLarge();  // At smaller sizes, table of contents is hidden
   }));
 
   it('has basic table of contents in the test page', util.itHelper(async () => {
-    expect(await tocPage.getLevelOneItemText()).to.equal('Test Page');
-    expect(await tocPage.getLevelTwoItemsText()).to.include.members([
+    expect(await tocPage.getLevelOneItemText(), 'Level one item text').to.equal('Test Page');
```

Comment:
What is this doing?

Reply:
it's getting all of the matching (by css class) items and compares the text.